test/parallel/test-console.js

```diff
@@ -115,7 +115,7 @@ assert.strictEqual("{ foo: 'bar', inspect: [Function: inspect] }\n",
                    strings.shift());
 assert.strictEqual("{ foo: 'bar', inspect: [Function: inspect] }\n",
                    strings.shift());
-assert.notEqual(-1, strings.shift().indexOf('foo: [Object]'));
+assert.notStrictEqual(-1, strings.shift().indexOf('foo: [Object]'));
```

Comment:
For readability, might be good to change this to `assert(strings.shift().includes('foo: [Object]'));`